elements/pl-checkbox/pl-checkbox.py

```diff
@@ -400,6 +408,7 @@ def grade(element_html, data):
     submitted_keys = data['submitted_answers'].get(name, [])
     correct_answer_list = data['correct_answers'].get(name, [])
     correct_keys = [answer['key'] for answer in correct_answer_list]
+    feedback = [option['feedback'] for option in data['params'][name] if option['key'] in submitted_keys]
```

Comment:
We need to handle the situation where the `feedback` key doesn't exist - this will be the case for any variant generated before this change gets deployed.

```suggestion
    feedback = [option.get('feedback', None) for option in data['params'][name] if option['key'] in submitted_keys]
```